SocialFilter/sections/general_elemhide.txt

```diff
@@ -15,17 +15,17 @@
 !+ NOT_OPTIMIZED
 ~facebook.com##a[href^="fb-messenger://share"]
 !+ NOT_OPTIMIZED
-~twitter.com##a[href^="https://twitter.com/intent/tweet?"]
+~twitter.com,~x.com##a[href^="https://x.com/intent/tweet?"]
 !+ NOT_OPTIMIZED
-~twitter.com##a[href^="https://twitter.com/intent/follow?"]
+~twitter.com,~x.com##a[href^="https://x.com/intent/follow?"]
```

Comment:
Whether twitter or x, should this generically be blocked in the first place? We don't block follow buttons except for specific context, right?

Reply:
@Alex-302 The first hit in publicwww showed FP `https://am-net.jp/`

Reply:
Moved to annoyances (in this PR).

Reply:
You forgot excluding twitter and x.

Reply:
> You forgot excluding twitter and x. NVM, forgot your comment that not in use on X now.

I did not see `a` at all.



> But you have to move corresponding exception rules too.

Not sure if they important when the rules in Annoyances. Removed them 
```
rankingoo.net#@#a[href^="https://twitter.com/intent/follow?"]
suits-woman.jp#@#a[href^="https://twitter.com/intent/follow?"]
oricon.co.jp#@#a[href^="https://twitter.com/intent/follow?"]
taishu.jp#@#a[href^="https://twitter.com/intent/follow?"]
comss.ru#@#a[href^="https://twitter.com/intent/follow?"]
sn-jp.com,snjpn.net#@#a[href^="https://twitter.com/intent/follow?"]
```